ldaptor/protocols/ldap/ldapsyntax.py

```diff
@@ -244,7 +244,7 @@ def __str__(self):
         for key, values in lst:
             if key != 'objectClass':
                 a.append((key, values))
-        return ldif.asLDIF(self.dn, a)
+        return ldif.asLDIF(self.dn.toWire().decode('utf-8'), a)
```

Comment:
Chage in API, from DN Object to Unicode?

Reply:
Reverted the changes as in the cases above: ``ldif`` module functions convert ``self.dn`` to byte string.

Reply:
were these changes reverted?

Reply:
My bad, fixed it with the latest commits